src/get-started/java-binding.md

```diff
@@ -472,6 +481,63 @@ client.
 An example of API service tests can be found in
 [`ApiControllerTest`][apicontrollertest].
 
+## Using Libraries
+
+An Exonum service can use any third-party library as its dependency.
+
+The libraries, classes from which are used in Exonum public APIs,
```

Comment:
I would rewrite this part something like the following:

> From another side, Exonum comes with its dependencies, classes from which are used in Exonum public APIs. The following dependencies are provided by the framework and **must** be used as provided. They will not be changed in an incompatible way in a compatible Exonum release.
> 
> !!! note
> These dependencies do not have to be declared explicitly for your service 
> because it already depends on `exonum-java-binding-core` which
> has them as transitive dependencies.
> 
> - Exonum (exonum-java-binding-core, exonum-java-binding-common)
> - [Guice][Guice]
> - [Gson][gson]
> - [Vertx][vertx-web-docs] (vertx-web)
> - [Protobuf Java] (https://github.com/protocolbuffers/protobuf/tree/master/java)
> - [PF4J] (https://pf4j.org)
> 
> These libraries *must not* be packaged in its artifact.
> To achieve that the [`provided`] (https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope) Maven dependency scope should be used in the dependency declarations.
>